### ISSUE TITLE
Error when getting statuses from Pleroma instance.

### ISSUE THREAD

**Issue by Lynnesbian** [halcy/Mastodon.py#147]
Hi, I've encountered a strange error when trying to download toots from a Pleroma instance. The instance is https://catgirl.science, and this is the error:
```
Downloading and parsing tootsTraceback (most recent call last):
  File "./main.py", line 175, in <module>
    for t in get_toots_legacy(client, f.id):
  File "./main.py", line 123, in get_toots_legacy
    toots = client.fetch_next(toots)
  File "/usr/local/lib/python3.6/dist-packages/mastodon/Mastodon.py", line 1927, in fetch_next
    method = params['_pagination_method']
TypeError: 'NoneType' object is not subscriptable
```

Any idea how this could be fixed? Thanks in advance!

**Comment by halcy** [comment]
Well, the problem lies in that mastodon.py expects the pagination headers that mastodon sends to exist when paginating. Maybe there's a small difference here in how Pleroma does it? Does fetch_next work generally and only break for, say, the last page?

**Comment by jfmcbrayer** [comment]
Looks like this is the same issue (with paging) as #163.

**Comment by halcy** [comment]
Presumably fixed (as #163 is now)
